examples/multiple_test_pages/tests.js

```diff
@@ -0,0 +1,11 @@
+for (var i = 0; i < 200; i++) {
+  if (window.location.search === '?batch=1') {
```

Comment:
How would it influence "-P" flag? Would user still be able to specify number of parallel runs dynamically? Or is it completely unrelated?